packages/icons/src/stories/leg-icon-renderer.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import IconRenderer from "./icon-renderer";
+
+// import mock itinaries. These are all trip plan outputs from OTP.
+const bikeOnlyItinerary = require("@opentripplanner/itinerary-body/src/__mocks__/itineraries/bike-only.json");
+const bikeRentalItinerary = require("@opentripplanner/itinerary-body/src/__mocks__/itineraries/bike-rental.json");
+const eScooterRentalItinerary = require("@opentripplanner/itinerary-body/src/__mocks__/itineraries/e-scooter-rental.json");
+const parkAndRideItinerary = require("@opentripplanner/itinerary-body/src/__mocks__/itineraries/park-and-ride.json");
+const tncTransitTncItinerary = require("@opentripplanner/itinerary-body/src/__mocks__/itineraries/tnc-transit-tnc.json");
+const walkOnlyItinerary = require("@opentripplanner/itinerary-body/src/__mocks__/itineraries/walk-only.json");
+const walkTransitWalkItinerary = require("@opentripplanner/itinerary-body/src/__mocks__/itineraries/walk-transit-walk.json");
+const walkTransitWalkTransitWalkItinerary = require("@opentripplanner/itinerary-body/src/__mocks__/itineraries/walk-transit-walk-transit-walk.json");
+
+const exampleLegs = [
+  { leg: bikeOnlyItinerary.legs[0], type: "Personal bike" },
+  { leg: bikeRentalItinerary.legs[1], type: "Bike rental" },
+  { leg: walkTransitWalkTransitWalkItinerary.legs[1], type: "Bus" },
+  {
+    leg: eScooterRentalItinerary.legs[1],
+    type: "Micromobility (from Razor company)"
+  },
+  { leg: parkAndRideItinerary.legs[0], type: "Park & Ride" },
+  { leg: tncTransitTncItinerary.legs[0], type: "TNC (Uber)" },
+  { leg: walkTransitWalkItinerary.legs[1], type: "Tram" },
+  { leg: walkOnlyItinerary.legs[0], type: "Walk" }
+];
```

Comment:
The leg is needed because it is an input into a LegIcon component.

I'm not sure how this related to #27. I believe we may have already resolved it by have a TriMetLegIcon, ClassicLegIcon and StandardLegIcon classes.

Reply:
Might be for another discussion... if we addressed #27 you wouldn't have to load all of these itineraries just to extract one leg and ultimately the mode for that leg.